ddl/ddl_tiflash_api.go

```diff
@@ -57,22 +57,156 @@ type TiFlashReplicaStatus struct {
 	IsPartition    bool
 }
 
+// TiFlashTick is type for backoff threshold.
+type TiFlashTick float64
+
+// PollTiFlashBackoffElement records backoff for each TiFlash Table.
+// When `Counter` reached `Threshold`, `Threshold` shall grow.
+type PollTiFlashBackoffElement struct {
+	Counter      int
```

Comment:
What is this counter? Can you explain it?

Reply:
I will compose a clearer comment